examples/shader-coverage-image-pipeline/main.cpp

```
@@ -0,0 +1,521 @@
+              << (enableCoverage ? " (coverage on)\n" : " (no coverage)\n");
+    auto shader = compileShader(enableCoverage);
+
+    uint32_t counterCount = 0;
+    if (enableCoverage)
+    {
+        counterCount = shader.coverageMetadata->getCounterCount();
+        std::cout << "coverage counter count: " << counterCount << "\n";
+    }
+
+    vkdemo::Context ctx;
+    ctx.init();
```

Cppcheck (2.20.0)

[error] Unhandled exception thrown in function that is an entry point.

(throwInEntryPoint)